stockflux-securities-master/src/components/ToolTip.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./ToolTip.css";
+import PropTypes from "prop-types";
+
+const ToolTip = ({ children, message, className }) => {
```

Comment:
As far as I can see the only props that Tooltip can take are `children` and `message`, aka no `className`, so I think it can be removed.